packages/core/src/products/details/client/__tests__/getProductOutfits.test.js

```diff
@@ -0,0 +1,45 @@
+import { getProductOutfits } from '..';
+import { mockOutfits, mockProductId } from 'tests/__fixtures__/products';
+import client from '../../../../helpers/client';
+import fixtures from '../__fixtures__/getProductOutfits.fixtures';
+import moxios from 'moxios';
+
+describe('getProductOutfits', () => {
+  const expectedConfig = undefined;
+  const spy = jest.spyOn(client, 'get');
+
+  beforeEach(() => {
+    moxios.install(client);
+    jest.clearAllMocks();
+  });
+
+  afterEach(() => moxios.uninstall(client));
+
+  it('should handle a client request successfully', async () => {
+    const response = mockOutfits;
+
+    fixtures.success({
+      id: mockProductId,
+      response,
+    });
+
+    expect.assertions(2);
```

Comment:
No need for `expect.assertions` as the test is awaiting the promise from the expect. If the promise fails, the test will fail.

Reply:
We have followed the same logic of existing similar tests, for consistency sake.

Reply:
I understand, but it could be removed as it is not necessary to avoid having to refactor this test as well when we fix those tests.

Reply:
Understood. Will make the change.